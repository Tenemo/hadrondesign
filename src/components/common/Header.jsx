import React from 'react';
import { Link, NavLink } from 'react-router-dom'; // eslint-disable-line no-unused-vars

const Header = () => {
    return (
        <div className="navbar navbar-expand-sm navbar-light bg-light navbar-nav">
            <Link to="/" className="navbar-brand">HD</Link>
            <NavLink exact to="/" class="nav-item">
                <div className="nav-link">About</div>
            </NavLink>
            <NavLink to="/portfolio" class="nav-item">
                <div className="nav-link">Portfolio</div>
            </NavLink>
            <NavLink to="/game" class="nav-item">
                <div className="nav-link">Game</div>
            </NavLink>
            <NavLink to="/contact" class="nav-item">
                <div className="nav-link">Contact</div>
            </NavLink>
        </div>
    );
};

export default Header;