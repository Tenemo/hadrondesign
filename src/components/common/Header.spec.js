import React from 'react';
import Header from './Header';

describe('Header', () => {
    it('matches snapshot', () => {
        const wrapper = shallow(pug`
            Header
        `);
        expect(wrapper).toMatchSnapshot();
    });
});