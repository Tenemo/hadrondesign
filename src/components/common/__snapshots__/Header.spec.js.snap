// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<div
  className="navbar navbar-expand-sm navbar-light bg-light navbar-nav"
>
  <Link
    className="navbar-brand"
    replace={false}
    to="/"
  >
    HD
  </Link>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    class="nav-item"
    exact={true}
    to="/"
  >
    <div
      className="nav-link"
    >
      About
    </div>
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    class="nav-item"
    to="/portfolio"
  >
    <div
      className="nav-link"
    >
      Portfolio
    </div>
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    class="nav-item"
    to="/game"
  >
    <div
      className="nav-link"
    >
      Game
    </div>
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    class="nav-item"
    to="/contact"
  >
    <div
      className="nav-link"
    >
      Contact
    </div>
  </NavLink>
</div>
`;
